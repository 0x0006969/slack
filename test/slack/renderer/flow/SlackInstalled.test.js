const { SlackInstalled } = require('../../../../lib/slack/renderer/flow');

describe('SlackInstalled', () => {
  test('works', async () => {
    const message = new SlackInstalled();
    expect(message.toJSON()).toMatchSnapshot();
  });
});
