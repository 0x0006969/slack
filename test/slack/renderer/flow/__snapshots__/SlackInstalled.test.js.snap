// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SlackInstalled works 1`] = `
Object {
  "attachments": Array [
    Object {
      "color": "#24292f",
      "text": "You've successfully installed GitHub on this Slack workspace :tada:
To subscribe a channel to a repository use the following slash command:
/github subscribe owner/repository
",
    },
  ],
}
`;
